apps/web/src/components/Composer/Actions/CollectSettings/AmountConfig.tsx

```diff
@@ -17,6 +17,12 @@ const AmountConfig: FC<AmountConfigProps> = ({
   allowedTokens,
   setCollectType
 }) => {
+  const [options] = useState(
+    allowedTokens?.map((token) => ({
+      label: token.name,
+      value: token.contractAddress
+    }))
+  );
```

Comment:
its not a good idea to have state everywhere 😕

Reply:
It causes problems with selection otherwise, the only other option is to move options outside of RFC but that causes problems where the option.selected are dependent on other props